datadog-monitors-downtimeschedule-handler/docs/README.md

```diff
@@ -0,0 +1,139 @@
+# Datadog::Monitors::DowntimeSchedule
+
+Datadog Downtime Schedule 0.0.1
+
+## Syntax
+
+To declare this entity in your AWS CloudFormation template, use the following syntax:
+
+### JSON
+
+<pre>
+{
+    "Type" : "Datadog::Monitors::DowntimeSchedule",
+    "Properties" : {
+        "<a href="#displaytimezone" title="DisplayTimezone">DisplayTimezone</a>" : <i>String</i>,
+        "<a href="#message" title="Message">Message</a>" : <i>String</i>,
+        "<a href="#mutefirstrecoverynotification" title="MuteFirstRecoveryNotification">MuteFirstRecoveryNotification</a>" : <i>Boolean</i>,
+        "<a href="#scope" title="Scope">Scope</a>" : <i>String</i>,
+        "<a href="#notifyendstates" title="NotifyEndStates">NotifyEndStates</a>" : <i>[ String, ... ]</i>,
+        "<a href="#notifyendtypes" title="NotifyEndTypes">NotifyEndTypes</a>" : <i>[ String, ... ]</i>,
+        "<a href="#monitoridentifier" title="MonitorIdentifier">MonitorIdentifier</a>" : <i><a href="monitoridentifier.md">MonitorIdentifier</a></i>,
+        "<a href="#schedule" title="Schedule">Schedule</a>" : <i><a href="schedule.md">Schedule</a></i>
+    }
+}
+</pre>
+
+### YAML
+
+<pre>
+Type: Datadog::Monitors::DowntimeSchedule
+Properties:
+    <a href="#displaytimezone" title="DisplayTimezone">DisplayTimezone</a>: <i>String</i>
+    <a href="#message" title="Message">Message</a>: <i>String</i>
+    <a href="#mutefirstrecoverynotification" title="MuteFirstRecoveryNotification">MuteFirstRecoveryNotification</a>: <i>Boolean</i>
+    <a href="#scope" title="Scope">Scope</a>: <i>String</i>
+    <a href="#notifyendstates" title="NotifyEndStates">NotifyEndStates</a>: <i>
+      - String</i>
+    <a href="#notifyendtypes" title="NotifyEndTypes">NotifyEndTypes</a>: <i>
+      - String</i>
+    <a href="#monitoridentifier" title="MonitorIdentifier">MonitorIdentifier</a>: <i><a href="monitoridentifier.md">MonitorIdentifier</a></i>
+    <a href="#schedule" title="Schedule">Schedule</a>: <i><a href="schedule.md">Schedule</a></i>
+</pre>
+
+## Properties
+
+#### DisplayTimezone
```

Comment:
Is there a reason you used H4 for this and the other headings in this section? H3 is also available.

Reply:
This is auto generated using aws cfn tools so unfortunately we don't have control over it.